### PR TITLE
Dark Theme enabling and Fixes

### PR DESCRIPTION
**Pull request by alioso**

### CODE REVIEW

File: src/components/PrimaryNav/PrimaryNav.tsx [hunk 1]
```diff
@@ -139,7 +139,7 @@ const styles = (theme: Theme & Linode.Theme): StyleRules => ({
     width: 'calc(100% - 20px)',
     justifyContent: 'center',
     // hidding for now - replace with flex
```
Comment (asauber) [inline src/components/PrimaryNav/PrimaryNav.tsx]: Keep comments up to date : )

**Comment by patthiel** [comment]
small issue with radios on Add a domain:

screen shot 2018-08-14 at 9 24 46 am

**Comment by Jskobos** [comment]
The text contrast in light theme is hard to read here:

screen shot 2018-08-14 at 9 48 59 am

**Comment by alioso** [comment]
fixed all items. thx for the feedback!

**Comment by patthiel** [comment]
Not a blocker, but the "Provisioning/Booting" text contrast should be higher. Will create a separate ticket
screen shot 2018-08-14 at 12 55 41 pm

**Pull request merged**